packages/xo-server-backup-reports/src/index.js

```diff
@@ -145,73 +246,181 @@ class BackupReportsXoPlugin {
     this._xo.on('job:terminated', this._report)
   }
 
-  test({ runId }) {
-    return this._backupNgListener(undefined, undefined, undefined, runId)
+  async test({ runId }) {
+    const xo = this._xo
+
+    const log = await xo.getBackupNgLogs(runId)
+    if (log === undefined) {
+      throw new Error(`no log found with runId=${JSON.stringify(runId)}`)
+    }
+
+    const job = await xo.getJob(log.jobId)
+    return job.type === 'backup'
+      ? this._backupNgListener(log, undefined, runId, true)
+      : this._metadataBackupListener(log, undefined, runId, true)
```

Comment:
`test` must be as close as possible as the normal behavior, you should use `this._report`.